packages/file_selector/file_selector_windows/example/lib/save_text_page.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+import 'package:flutter/material.dart';
+
+/// Page for showing an example of saving with file_selector
```

Comment:
nit: for consistency use 'Screen' here or 'Page' across the board. Not sure which I prefer, personally -- both seem fine :)